<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinancesWidget</class>
 <widget class="QWidget" name="FinancesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="TopFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Title">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Finances</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="BottomFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="ListFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="ListFrameLine">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>688</width>
              <height>205</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="InputsFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QFormLayout" name="Inputs">
           <item row="0" column="0">
            <widget class="QLineEdit" name="TitleInput"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="TitleLabel">
             <property name="text">
              <string>Title</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="DescriptionInput"/>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="DescriptionLabel">
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="PriceInput"/>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="PriceLabel">
             <property name="text">
              <string>Price (Â£)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="DateInput"/>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="DateLabel">
             <property name="text">
              <string>Date (yyyy-mm-dd)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="InputsButtonsLine">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="CancelButton">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CreateButton">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FinancesWidget</receiver>
   <slot>CancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CreateButton</sender>
   <signal>clicked()</signal>
   <receiver>FinancesWidget</receiver>
   <slot>CreateButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
